qaSchemaAndETL.sql

```diff
@@ -77,34 +77,55 @@ CREATE TABLE photos (
 -- ---
 
 -- ALTER TABLE questions ADD FOREIGN KEY (product_id) REFERENCES product_index (id);
-ALTER TABLE answers ADD FOREIGN KEY (question_id) REFERENCES questions (id);
-ALTER TABLE photos ADD FOREIGN KEY (answer_id) REFERENCES answers (id);
+ALTER TABLE answers ADD FOREIGN KEY (question_id) REFERENCES questions (question_id) ON DELETE CASCADE;
+ALTER TABLE photos ADD FOREIGN KEY (answer_id) REFERENCES answers (id) ON DELETE CASCADE;
 
 -- ---
 -- ETL Questions
 -- ---
 
-\COPY questions (id, product_id, question_body, epoch_date, asker_name, asker_email, reported, question_helpfulness) from '/Users/coryellerbroek/Desktop/HackReactor/sdc-questions-and-answers/datasets/questions.csv' DELIMITER ',' CSV HEADER;
+\COPY questions (question_id, product_id, question_body, epoch_date, asker_name, asker_email, reported, question_helpfulness) from '/Users/coryellerbroek/Desktop/HackReactor/sdc-questions-and-answers/datasets/sample_questions.csv' DELIMITER ',' CSV HEADER;
 
-UPDATE questions SET date = to_timestamp(floor(epoch_date / 1000));
+UPDATE questions SET question_date = to_timestamp(floor(epoch_date / 1000));
 
 ALTER TABLE questions DROP COLUMN epoch_date;
 
+CREATE INDEX question_helpfulness_index ON questions (question_helpfulness);
```

Comment:
I need to do this for my Schemas too, but I was going to wait until I did the stress testing to implement the ones causing trouble.... I'm expecting my meta queries to all be indexed.